### PR TITLE
feat: add a last N time range selector on project / projects pages

### LINKED ISSUE #2883: [persistence] time range selector for traces / spans

### PR DESCRIPTION
**Pull request by mikeldking**
resolves #2883

Adds a simple last N time range selector that defaults to 7 days across project pages.

https://github.com/Arize-ai/phoenix/assets/5640648/a77980c9-9923-447f-99fe-78cca817103a

### CODE REVIEW

File: app/src/pages/project/ProjectPage.tsx [hunk 1]
```diff
@@ -39,9 +45,34 @@ const mainCSS = css`
 
 export function ProjectPage() {
   const { projectId } = useParams();
+  const { timeRange } = useLastNTimeRange();
+  return (
+    <Suspense fallback={<Loading />}>
+      <ProjectPagContent
```
Comment (axiomofjoy) [inline app/src/pages/project/ProjectPage.tsx:51]: Type here.

File: app/src/pages/project/ProjectPage.tsx [hunk 2]
```diff
@@ -39,9 +45,34 @@ const mainCSS = css`
...
+        projectId={projectId as string}
+        timeRange={timeRange}
+      />
+    </Suspense>
+  );
+}
+
+export function ProjectPagContent({
```
Comment (axiomofjoy) [inline app/src/pages/project/ProjectPage.tsx:59]: Typo here

**Review by axiomofjoy — APPROVED** [review]
nice

File: app/src/pages/project/ProjectPage.tsx (hunk 1 above)
Comment (mikeldking) [reply app/src/pages/project/ProjectPage.tsx:51]: thanks

**Pull request merged**